protocol-designer/src/labware-ingred/actions/thunks.ts

```diff
@@ -111,7 +112,7 @@ export const duplicateLabware: (
       payload: {
         duplicateLabwareNickname,
         templateLabwareId,
-        duplicateLabwareId: uuid(),
+        duplicateLabwareId: uuid() + ':' + templateLabwareDefURI,
```

Comment:
this doesn't necessarily fix anything here but it is needed for multi tiprack support

Reply:
maybe using Template literals instead of `+`?

```suggestion
        duplicateLabwareId: `${uuid()}:${templateLabwareDefURI}`,
```